main/main.cpp

```diff
@@ -1179,28 +1179,28 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 
 	OS::get_singleton()->set_cmdline(execpath, main_args);
 
-	GLOBAL_DEF("rendering/quality/driver/driver_name", "Vulkan");
```

Comment:
It's nice to finally get rid of this "quality" subsection that had nothing to do with quality...